<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Board</class>
 <widget class="QDialog" name="Board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Board</string>
  </property>
  <widget class="QPushButton" name="CreateServer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>创建房间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JoinRoom">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>加入房间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartGame">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Regret">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveBoard">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdmitLoser">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>认输</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LeftTime">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TotalTime">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>510</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时限</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>490</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>总时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenBoard">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>打开残局</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
